pkg/gce-pd-csi-driver/controller.go

```diff
@@ -185,7 +185,7 @@ func (gceCS *GCEControllerServer) DeleteVolume(ctx context.Context, req *csi.Del
 		return nil, status.Error(codes.InvalidArgument, "DeleteVolume Volume ID must be provided")
 	}
 
-	volKey, err := common.VolumeIDToKey(volumeID)
+	volKey, _, err := common.VolumeIDToKey(volumeID)
```

Comment:
I think because we are not supporting creating volumes with partitions, we should not support delete volume with partitions.  You don't want to delete a disk that has multiple partitions on it because DeleteVolume got called for one of the partitions